dockerfile/cuda12.2.dockerfile

```diff
@@ -122,7 +122,15 @@ RUN cd /tmp && \
     rm -rf /tmp/nccl
 
 ENV PATH="${PATH}" \
-    LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}" \
+    LD_LIBRARY_PATH="/usr/local/lib:\
+/opt/hpcx/nccl_rdma_sharp_plugin/lib:\
```

Comment:
is it possible to append `source /opt/hpcx/hpcx-init.sh && hpcx_load` to `~/.bashrc` when installing hpcx? paths here may be changed in future hpcx versions

Reply:
Good point. However, each time docker run is executed, it mounts a different workspace to /root using -v {{ workspace }}:/root, which deletes all the original files in /root. So, changed to add this file to /opt/superbench at first, and restore the files during docker exec. What do you think of this?

Reply:
I forgot about it. how about appending to system wide `/etc/bash.bashrc`? [it's loaded before `~/.bashrc`](https://askubuntu.com/a/815083/1776723)

Reply:
Make sense to me and verified.